image_classification/utils/utils.py

```diff
@@ -0,0 +1,121 @@
+import os
+import numpy as np
+import torch
+from fastai.vision import *
+from image_classification.models.custom_resnet import *
```

Comment:
import as `from image_classification.models import custom_resnet` - why? see the comment below